src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml

```diff
@@ -1,168 +1,84 @@
+                    <wasm:Setter Target="PersonPictureImage.Source" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ActualImageBrush.ImageSource}" />
+                  </VisualState.Setters>
+                </VisualState>
+                <!-- Visual State when no Photo is available (but there are initials).-->
+                <VisualState x:Name="Initials" />
+                <VisualState x:Name="NoPhotoOrInitials">
+                  <VisualState.Setters>
+                    <Setter Target="InitialsTextBlock.FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />
+                    <!-- Placeholder symbol for 'Contact' (https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font) -->
+                    <Setter Target="InitialsTextBlock.Text" Value="&#xE77B;" />
+                  </VisualState.Setters>
+                </VisualState>
+                <!-- Visual State when in Group mode -->
+                <VisualState x:Name="Group">
+                  <VisualState.Setters>
+                    <Setter Target="InitialsTextBlock.FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />
+                    <!-- Placeholder symbol for 'People' (https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font) -->
+                    <Setter Target="InitialsTextBlock.Text" Value="&#xE716;" />
+                  </VisualState.Setters>
+                </VisualState>
+              </VisualStateGroup>
+              <VisualStateGroup x:Name="BadgeStates">
+                <VisualState x:Name="NoBadge" />
+                <VisualState x:Name="BadgeWithoutImageSource">
+                  <VisualState.Setters>
+                    <Setter Target="BadgeGrid.Visibility" Value="Visible" />
+                  </VisualState.Setters>
+                </VisualState>
+                <VisualState x:Name="BadgeWithImageSource">
+                  <VisualState.Setters>
+                    <Setter Target="BadgeGrid.Visibility" Value="Visible" />
+                    <Setter Target="BadgingEllipse.Opacity" Value="{ThemeResource PersonPictureEllipseBadgeImageSourceStrokeOpacity}" />
+                    <Setter Target="BadgingEllipse.Fill">
+                      <Setter.Value>
+                        <ImageBrush x:Name="BadgeImageBrush" Stretch="UniformToFill" />
+                      </Setter.Value>
+                    </Setter>
+                  </VisualState.Setters>
+                </VisualState>
+              </VisualStateGroup>
+            </VisualStateManager.VisualStateGroups>
+            <Ellipse Fill="{ThemeResource PersonPictureEllipseFillThemeBrush}" Stroke="{ThemeResource PersonPictureEllipseFillStrokeBrush}" StrokeThickness="{ThemeResource PersonPictureEllipseStrokeThickness}" Width="{TemplateBinding Width}" Height="{TemplateBinding Height}" />
```

Comment:
I think we should be reverting this and keep the `TemplateBinding` code? @MartinZikmund thoughts?

Reply:
You mean reverting the workaround?

Reply:
Ah yes, we should include it, Microsoft omitted it from WinUI3 probably by accident